### ISSUE TITLE
Bpython can't pickle

### ISSUE THREAD

**Issue by jkbbwr** [bpython/bpython#664]
Smallest test case
```
import pickle
class Test(object):
     pass
pickle.dumps(Test())
```
Traceback
```
Traceback (most recent call last):
  File "<input>", line 1, in <module>
    pickle.dumps(Test())
_pickle.PicklingError: Can't pickle <class '__console__.Test'>: import
 of module '__console__' failed
```

**Comment by sebastinas** [comment]
This is a bug in the curtsies frontend. cli works.

**Comment by sebastinas** [comment]
To be more precise: it only occurs with `python -m bpython.curtsies`. `python -m bpython` works as expected.

**Comment by sebastinas** [comment]
@jkbbwr Could you please test if the issues still exists after f189da1?

**Comment by sebastinas** [comment]
So let's consider it fixed with f189da1.
